easybuild/easyconfigs/p/plc/plc-2.0-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,86 @@
+easyblock = 'Waf'
+
+name = 'plc'
+version = '2.0'
+
+homepage = 'http://pla.esac.esa.int/pla/#home'
+
+description = """
+plc is the public Planck Likelihood Code.
+It provides C and Fortran libraries that allow 
+users to compute the log likelihoods of the temperature, 
+polarization, and lensing maps.  Optionally, it also provides a python version of this library, 
+as well as tools to modify the predetermined options for some likelihoods 
+(e.g. changing the high-ell and low-ell lmin and lmax values of the temperature).
+"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['http://pla.esac.esa.int/pla/aio']
+sources = ['product-action?COSMOLOGY.FILE_ID=COM_Likelihood_Code-v2.0_R2.00.tar.bz2']
+
+patches = [
+    'plc-2.0-fix-build.patch',		 # workarounds fixing waf build behavior in EB environment
+    'plc-2.0-fix-openmp-flags.patch',	 # add proper (-fopenmp) openmp flags for icc and ifort
```

Comment:
please use `%(version)s` rather than hardcoding the verison, same in `sources`

Reply:
using buildininstalldir = 'true' is seems to be slightly cleaner than fixing the install procedure with patches, so i did so.